client/post-editor/editor-status-label/index.jsx

```diff
@@ -73,7 +81,10 @@ export default React.createClass( {
 				aria-label={ this.translate( 'Show advanced status details' ) }
 				aria-pressed={ !! this.props.advancedStatus }
 			>
-				<Gridicon icon="cog" size={ 16 } />
+				{ this.props.advancedStatus
+					? <Gridicon icon="chevron-up" size={ 18 } />
```

Comment:
The design was done before that change went in production. So it's misaligned, and it should be a cog. Sorry about that.